### ISSUE TITLE
ADM doesn't switch theme for Outlook

### ISSUE THREAD

**Issue by anishre** [AutoDarkMode/Windows-Auto-Night-Mode#487]
### Description

ADM doesn't switch dark theme on/off for Outlook but does for other MS365 apps.

### Expected Behavior

ADM should change theme for all Office apps according to time switch, but it doesn't do any toggling for outlook app. It has to be changed manually.

### Log Data

[service.log](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/files/9124517/service.log)

### Operating System

Windows 10 Pro Version 21H2, OS build 19044.1826

### Commit Hash and Version

- Commit: `5e6ea8a`
- Service: `10.1.0.10`
- Updater: `2.4.2`
- App: `10.0.1.0`
- Shell: `1.3.1.0`
- .Net: `6.0.0`

### ADM comparison for Excel and Outlook at the same time
![image](https://user-images.githubusercontent.com/39543074/179320724-d11cba53-bd9f-40ac-b575-74471acfeda6.png)

**Comment by Armin2208** [comment]
Have you activated "follow system theme" at file -> office account -> office theme -> use system theme

**Comment by anishre** [comment]
Yeah, I had that selected but after every "time based toggle" it would be stuck on the previous theme. For instance I set it to follow system theme when its dark (night time) and when system theme switches in the morning, theme for Outlook is still dark. Happens vice versa as well.

**Comment by Spiritreader** [comment]
ADM's Office Switch functionality is for Office versions that don' have the "use system theme" setting in office.
It will not work with current office versions, hence it is marked as legacy

**Comment by Armin2208** [comment]
What is more clear?
The office version years?
The legacy mark?
Or something like (not 365) :D

**Comment by KeeLee2005** [comment]
But Microsoft® Outlook®  Microsoft 365MSO 2208 Build 16.0.15601.20072 doesn't have this problem.

**Comment by samuelT2** [comment]
Same problem with Outlook 2210 Build 16.0.15726.20070 64 Bit (MS 365):

**Scenario**

Pre-condition: Office design is set to "Use system settings" (eg. in Outlook).

1. Using the computer during the day with the bright design
2. ADM triggers dark mode sometime during the evening, all designs go dark (Windows & Office). 
3. The next morning, computer boots up, ADM triggers light mode correctly.
4. Office products behave differently:
- Outlook theme setting is on "black"
- Word theme setting is on "colorful"

Why does the office setting change anyway? Ideally, it would stay on "Use system settings".

I can provide more debug info if needed.

**Comment by Spiritreader** [comment]
The office setting does not work for O365 and can't be used. If you do you get unexpected behavior.

**Comment by samuelT2** [comment]
Thanks @Spiritreader, I'll deactivate the Microsoft Office setting in ADM.

**Comment by Armin2208** [comment]
no chance because it's an web app

**Comment by emberaholdon** [comment]
> Any chance of supporting One Outlook (next generation) Dark Mode? It has a toggable setting in settings, but there's no option to sync with system theme.
> 
> https://www.microsoft.com/store/productId/9NRX63209R7B

The current version of Outlook (new) (packed with Windows 11 23H2) supports auto theme switching and working fine on my computer.

If you use PWA apps just install the Dark Reader extension for Edge.

**Comment by emberaholdon** [comment]
> Any chance of supporting One Outlook (next generation) Dark Mode? It has a toggable setting in settings, but there's no option to sync with system theme.
> 
> https://www.microsoft.com/store/productId/9NRX63209R7B

The current version of Outlook (new) (packed with Windows 11 23H2) supports auto theme switching and working fine on my computer.

If you use PWA apps just install the Dark Reader extension for Edge.

**Comment by luke7101** [comment]
I have a different question around this theme:
I also noticed that the current Office 365 is correctly switching theme following the system settings if "Use system settings" is set in the Office apps. The current effect is like toggling between the "White theme" for the day and the "Black theme" for the night.
My question is: would it be possible to toggle between "White" and "Dark grey" instead of "Black" ? (or suggest a workaround if there is one ?)

**Comment by emberaholdon** [comment]
I don't think there is one at the moment.

You can set up a custom script that modifies the registry, but the apps need to be restarted, they don't change while running.

The key is located at: HKEY_CURRENT_USER\Software\Microsoft\Office\16.0\Common

Set "UI Theme" (DWORD) to:

3: Dark Grey
4: Black
5: White
6: Use system setting
7: Colourful

**Comment by Jay-o-Way** [comment]
Is this issue still relevant? (I think not?)
